doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,184 @@
+</table>
+
+
+## Known Issues
+
+The v0.9.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support</td>
+<td valign="top">AIX</td>
+<td valign="top">Inability to diagnose problems with the VM, garbage collector, or JIT.</td>
+<td valign="top">None</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/54">#54</a></td>
+<td valign="top">Signal handling</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some signals not being handled correctly by the VM.</td>
+<td valign="top">None</td>
+</tr>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/479">#479</a></td>
+<td valign="top">Non-compressed references VM builds not available</td>
+<td valign="top">AIX, Windows, Linux on Z, Linux on Power</td>
+<td valign="top">Heaps larger than 57GB are not supported.</td>
+<td valign="top">Manual builds are possible by following our <a href="https://github.com/eclipse/openj9/blob/master/buildenv/Build_Instructions_V8.md">detailed build instructions</a>.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1938">#1938</a></td>
+<td valign="top">Default shared classes cache size (16MB) is too small on Java 8 64-bit binaries.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Poorer performance for applications that could benefit from a larger cache size.</td>
+<td valign="top">Manually set the shared cache size to at least 60 MB when you start your application (<code>-Xscmx60m</code>).</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1252">#1252</a></td>
+<td valign="top">File <code>ibmjvmti.h</code> not found in OpenJ9 JDK</td>
+<td valign="top">All platforms</td>
+<td valign="top">JVMTI extensions cannot be compiled.</td>
+<td valign="top">None.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1252">#1252</a></td>
+<td valign="top">GPU acceleration with `java.util.Array.sort()` is not supported</td>
+<td valign="top">Linux and Windows platforms</td>
+<td valign="top">GPU acceleration on `sort()` operations is not available.</td>
+<td valign="top">None, although `com.ibm.gpu.Maths` can be used directly.</td>
```

Comment:
This will not work: a `FileNotFoundException` will be thrown with respect to `SortKernels.fatbin` on a system with any recognized GPUs.

Reply:
More explanation required, please! Are you referring to com.ibm.gpu.Maths? What is SortKernels.fatbin?!

Reply:
Yes, com.ibm.gpu.Maths will (when one or GPUs are available) try to load SortKernels.fatbin (which is expected to contain the GPU sorting support code): the problem is that file is not created by the OpenJ9 build process currently, the result being a FileNotFoundException. We should not be advising use of Maths.sortArray() as an alternative to Arrays.sort() (at least until this issue is resolved).

Reply:
i.e. please remove "although `com.ibm.gpu.Maths` can be used directly."

Reply:
Thanks - done!